File: Modul 4/PRAK404/PRAK304-2310817310011-MuhammadRizky.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    inputPilihan = float(input("Masukkan Pilihan : "))

    if inputPilihan == 5:
            print("Terimakasih, telah menggunakan kalkulator Muhammad Rizky\n")
            break
    elif inputPilihan > 5 or inputPilihan <= 0:
           print("Input anda salah, silahkan coba lagi")
    else:
           inputPertama =float(input("Masukkan nilai pertama : "))
           inputKedua = float(input("Masukkan nilai kedua : "))

           if inputPilihan == 1:
                  hasil = inputPertama + inputKedua
                  print("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f" % (inputPertama, inputKedua,  hasil))

           elif inputPilihan == 2:
                  hasil = inputPertama - inputKedua
                  print("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f" % (inputPertama, inputKedua,  hasil))

           elif inputPilihan == 3:
                  hasil = inputPertama * inputKedua
                  print("Hasil perkalian antara %.2f dengan %.2f adalah %.2f" % (inputPertama, inputKedua,  hasil))

           elif inputPilihan == 4:
                  hasil = inputPertama / inputKedua
                  print("Hasil pembagian antara %.2f dengan %.2f adalah %.2f" % (inputPertama, inputKedua,  hasil))

File: Modul 4/PRAK404/PRAK304-2310817310011-MuhammadRizky.c
#include <stdio.h>

int main() {
    int inputPilihan;
    float inputPertama, inputKedua, hasil;

    while (1) {
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan: ");
        scanf("%d", &inputPilihan);

        if (inputPilihan == 5) {
            printf("Terimakasih, telah menggunakan kalkulator Muhammad Rizky\n");
            break;
        } else if (inputPilihan >= 1 && inputPilihan <= 4) {
            printf("Masukkan nilai pertama: ");
            scanf("%f", &inputPertama);
            printf("Masukkan nilai kedua: ");
            scanf("%f", &inputKedua);

            switch (inputPilihan) {
                case 1:
                    hasil = inputPertama + inputKedua;
                    printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", inputPertama, inputKedua, hasil);
                    break;
                    
                case 2:
                    hasil = inputPertama - inputKedua;
                    printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", inputPertama, inputKedua, hasil);
                    break;
                case 3:
                    hasil = inputPertama * inputKedua;
                    printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", inputPertama, inputKedua, hasil);
                    break;
                case 4:
                    hasil = inputPertama / inputKedua;
                    printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", inputPertama, inputKedua, hasil);
                    break;
                    
            }
        } else {
            printf("Input anda salah, silahkan coba lagi\n");
        }
    }

    return 0;
}
